4.4 execution plan summary (#1352)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc

|
|

| <<query-plan-create-nodes---relationships,Create>>
| Creates nodes and relationships.
|
| label:yes[]
|

| <<query-plan-create-index,CreateIndex>>
| Creates an index for either nodes or relationships.
|
| label:yes[]
|

| <<query-plan-create-node-key-constraint,CreateNodeKeyConstraint>>

> | Creates a node key constraint on a set of properties for all nodes having a certain label.

Reviewer: ```suggestion
| Creates a node key constraint on a set of properties for all nodes with a certain label.
```

|
|

| <<query-plan-exhaustive-limit,ExhaustiveLimit>>
a|

> The `ExhaustiveLimit` operator is similar to the `Limit` operator, but will always exhaust the input.

Reviewer: ```suggestion
The `ExhaustiveLimit` operator is similar to the `Limit` operator, but always exhausts the input.
```

|
|

| <<query-plan-optional,Optional>>
| Yields a single row with all columns set to `null` if no data is returned by its source.
|
|
|

| <<query-plan-optional-expand-all,OptionalExpand(All)>>
| Traverses relationships from a given node, producing a single row with the relationship and end node set to `null` if the predicates are not fulfilled.
|
|
|

| <<query-plan-optional-expand-into,OptionalExpand(Into)>>

> | Traverses all relationships between two nodes, producing a single row with the relationship and end node set to `null` if no matching relationships are found (the start node will be the node with the smallest degree).

Reviewer: ```suggestion
| Traverses all relationships between two nodes, producing a single row with the relationship and end node set to `null` if no matching relationships are found (the start node is the node with the smallest degree).
```
